packages/contracts/src/utils/constants.ts

```diff
@@ -19,6 +19,13 @@ const TESTRPC_PRIVATE_KEYS_STRINGS = [
 export const constants = {
     INVALID_OPCODE: 'invalid opcode',
     REVERT: 'revert',
+    LIB_BYTES_GT_ZERO_LENGTH_REQUIRED: 'Length must be greater than 0.',
+    LIB_BYTES_GTE_4_LENGTH_REQUIRED: 'Length must be greater than or equal to 4.',
+    LIB_BYTES_GTE_20_LENGTH_REQUIRED: 'Length must be greater than or equal to 20.',
+    LIB_BYTES_GTE_32_LENGTH_REQUIRED: 'Length must be greater than or equal to 32.',
+    LIB_BYTES_INDEX_OUT_OF_BOUNDS: 'Specified array index is out of bounds.',
+    ERC20_INSUFFICIENT_BALANCE: 'Insufficient balance to complete transfer.',
+    ERC20_INSUFFICIENT_ALLOWANCE: 'Insufficient allowance to complete transfer.',
```

Comment:
This error message seems generic enough to also apply to an ERC721 transfer... Does it not?

Reply:
ERC721 doesn't have the notion of balance. It's either 1 or 0. It would be better to have smth like `You must own that NFT to complete transfer`

Reply:
Intuitively it seems like it. But I pulled it directly from __ER20Token.sol__ and it looks like __ERC721Token.sol__ doesn't use it.